components/docs/Image.tsx

```diff
@@ -16,5 +16,5 @@ export default function DocImage(props: DocImageProps) {
     imageProps.src = `/images/${src}`;
   }
 
-  return <Image className="mb-4" {...imageProps} />;
+  return <Image {...imageProps} />;
```

Comment:
This was removed as it was causing weird cropping issues, even in production
![Screen Shot 2021-12-09 at 3 54 34 PM](https://user-images.githubusercontent.com/4967694/145474223-4a1661c5-025f-4905-94a2-f02660a5532a.png)
.